recipes-core/base-files/base-files_3.%.bbappend

```diff
@@ -1,8 +1,20 @@
-FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+FILESEXTRAPATHS_prepend_menderimage := "${THISDIR}/${PN}:"
 
 #TODO: ${@bb.utils.contains("MACHINE_FEATURES", "sd", "packagegroup-mender-sd", "", d)}
 #./meta/recipes-core/packagegroups/packagegroup-base.bb
-do_install_append () {
+
+do_patch_device() {
+    # Noop if we're not building a mender-image.
+    true
+}
+do_patch_device_menderimage() {
+    if [ -n "${MENDER_STORAGE_DEVICE_BASE}" ]; then
+        sed -i -e 's,/dev/mmcblk0p,${MENDER_STORAGE_DEVICE_BASE},g' ${S}/fstab
```

Comment:
The intention is to make the device configurable. But it should perhaps be made clearer here by replacing `@MENDER_STORAGE_DEVICE_BASE@` instead of `mmcblk0p`. I can change that.